code/modules/random_map/noise/magma.dm

```diff
@@ -1,44 +1,31 @@
-// This is basically filler at this point. Subsidence and all kinds of fun
-// hazards will be included when it is done.
-/datum/random_map/noise/volcanism
```

Comment:
Why are you deleting this?

Reply:
Does not work, so, I am replacing them with the basalt and lava ones.